recipes/litescope/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "litescope" %}
+{% set version = "2023.12" %}
+{% set sha256 = "dfc0ae42c87c315753332f2eab90570eae302ef1526c1cd1d12cae91159175b3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/enjoy-digital/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-build-isolation --no-deps
+  entry_points:
+    - litescope_cli = litescope.software.litescope_cli:main
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - litex >=2023.08
+    - migen
+
+test:
+  imports:
```

Comment:
Please add a call to pip check or a comment of why you are omitting it.

Reply:
same as `lite*` https://github.com/conda-forge/staged-recipes/pull/26117#discussion_r1581899783

Reply:
this work is under the umbrella of https://github.com/enjoy-digital/liteeth/issues/159 in case you're curious